--- /usr/share/php/XML/RPC2/Server/Input/RawPostData.php	2015-05-07 07:09:54.898259119 +0200
+++ /tmp/RawPostData.php	2015-05-07 07:15:00.662259119 +0200
@@ -74,6 +74,9 @@
      */
     public function readRequest()
     {
+        // Alexandre Espinosa Menor (2012-07-06): force this global
+        $GLOBALS['HTTP_RAW_POST_DATA'] = file_get_contents('php://input');
+
         if (!isset($this->input) && !isset($GLOBALS['HTTP_RAW_POST_DATA'])) throw new XML_RPC2_ConfigException('XML_RPC2_Server_Input_RawPostData requested but PHP config does not show GLOBALS[\'HTTP_RAW_POST_DATA\'] as available'); 
         if (!isset($this->input)) $this->input = $GLOBALS['HTTP_RAW_POST_DATA'];
 
